repos/system_upgrade/el7toel8/actors/storagescanner/tests/unit_test_storagescanner.py

```diff
@@ -0,0 +1,203 @@
+import os
+
```

Comment:
no clear why the half of the file is missing here. added back in my branch

Reply:
I see that the dropped tests are failing. However, not clear for me the reason why they have been dropped. Let's sync the next day.